src/Templates/NIPSWeb/live.twig

```diff
@@ -23,7 +23,7 @@
             <p>Connecting to server...</p>
             <div id="init-progressbar"></div>
         </div>
-        <div class="ui-dialog-buttonpane ui-widget-content ui-helper-hidden" id="init-overlay-fallback">
+        <div class="ui-dialog-buttonpane ui-widget-content hidden" id="init-overlay-fallback">
```

Comment:
The point that @Flump5281 was making is that it is still using the name behind the previous name...